[2024-05-13T03:32:18.035+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-13T03:32:18.045+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v5.first_task manual__2024-05-13T03:32:17.019574+00:00 [queued]>
[2024-05-13T03:32:18.048+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v5.first_task manual__2024-05-13T03:32:17.019574+00:00 [queued]>
[2024-05-13T03:32:18.048+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-13T03:32:18.052+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): first_task> on 2024-05-13 03:32:17.019574+00:00
[2024-05-13T03:32:18.055+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v5', 'first_task', 'manual__2024-05-13T03:32:17.019574+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpx2c9xfkk']
[2024-05-13T03:32:18.055+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=2612) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-05-13T03:32:18.056+0000] {standard_task_runner.py:91} INFO - Job 29: Subtask first_task
[2024-05-13T03:32:18.056+0000] {standard_task_runner.py:63} INFO - Started process 2613 to run task
[2024-05-13T03:32:18.069+0000] {task_command.py:426} INFO - Running <TaskInstance: our_first_dag_v5.first_task manual__2024-05-13T03:32:17.019574+00:00 [running]> on host f1ddc6e75993
[2024-05-13T03:32:18.096+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='mani' AIRFLOW_CTX_DAG_ID='our_first_dag_v5' AIRFLOW_CTX_TASK_ID='first_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-13T03:32:17.019574+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-13T03:32:17.019574+00:00'
[2024-05-13T03:32:18.096+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-13T03:32:18.101+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-13T03:32:18.102+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo Hello world, this is the first task!']
[2024-05-13T03:32:18.104+0000] {subprocess.py:86} INFO - Output:
[2024-05-13T03:32:18.105+0000] {subprocess.py:93} INFO - Hello world, this is the first task!
[2024-05-13T03:32:18.105+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-13T03:32:18.105+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-13T03:32:18.113+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v5, task_id=first_task, run_id=manual__2024-05-13T03:32:17.019574+00:00, execution_date=20240513T033217, start_date=20240513T033218, end_date=20240513T033218
[2024-05-13T03:32:18.120+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-13T03:32:18.127+0000] {taskinstance.py:3498} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2024-05-13T03:32:18.128+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
